dockerdiff

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+IMAGE1=$1
+IMAGE2=$2
+
+inspect() {
+    docker inspect $1 | jq ".[0]|del(.Id,.RepoTags,.RepoDigests,.Created)"
+}
+
+dpkgl() {
+    docker run --rm $1 dpkg -l
+}
+
+lslr() {
+    docker run --rm $1 bash -c 'find / -xdev -print0 | sort -z | xargs -0 ls -ld'
+}
+
+if ! diff -u <(inspect $IMAGE1) <(inspect $IMAGE2); then
```

Comment:
They could, but this is the only one that is needed. The others are just to 
give the user more information about what has changed.

If any of the other commands would fail, then this would already have failed,
that is if the `diff` of the `ls`, `md5sum`, or `dpkg` outputs would fail, then
the `inspect` diff would have failed.

What this is doing is just about giving more info to the user if it failed, so it
fails on the first one, but prints all the extra info.